tests/contrib/pymongo/test_pymongo_patch.py

```diff
@@ -21,11 +21,76 @@ class TestPymongoPatch(PatchTestCase.Base):
     __unpatch_func__ = unpatch
     __get_version__ = get_version
 
+    def _get_imports(self, pymongo):
+        if _VERSION >= (4, 9):
```

Comment:
Do don't need to the nest these imports in a function? We can just make them global?